CMakeLists.txt

```diff
@@ -3,7 +3,13 @@ project(poco_vendor VERSION "1.1.1")
 
 # Can work with poco 1.4.1p1 (earliest to use recursive mutexes on Linux)
 # 1.6.1 is the first version to ship with PocoConfigVersion.cmake
-find_package(Poco "1.6.1" COMPONENTS Foundation QUIET)
+if(NOT ROS2_FORCE_BUILD_ALL_VENDOR)
+  find_package(Poco "1.6.1" COMPONENTS Foundation QUIET)
+endif()
+
+if(ROS2_FORCE_LIBCXX)
```

Comment:
Rather than adding a custom CMake option `ROS2_FORCE_LIBCXX` to various packages how about changing this package to generically pass `CMAKE_CXX_FLAGS` to the external project?

Reply:
Good idea, I'll try that approach